vulnerablecode/urls.py

```diff
@@ -27,13 +27,17 @@
 from rest_framework.routers import DefaultRouter
 
 from vulnerabilities.api import PackageViewSet
+from vulnerabilities.api import VulnerabilityViewSet
 
 
-api_router = DefaultRouter()
-api_router.register(r'packages', PackageViewSet)
+api_router = DefaultRouter(trailing_slash=False)
```

Comment:
What's the reasoning behind overriding the default trailing_slash convention?

Reply:
I had to discuss this with you guys. 

For the `ListView` using trailing_slash the endpoints look  odd. The endpoints with queries look like :
`/api/vulnerabilities/?vulnerability_id=foo`

After the override you get the :
`/api/vulnerabilities?vulnerability_id=foo`

Not sure this matter tho, so I might be missing something here.

Reply:
I don't see the benefit of removing the trailing slash.

From https://www.django-rest-framework.org/api-guide/routers/

> Trailing slashes are conventional in Django, but are not used by default in some other frameworks such as Rails.

I think we should follow Django's conventions.

Reply:
:+1: